antsibull/data/docsite/plugins_by_collection.rst.j2

```diff
@@ -29,20 +29,35 @@ Collection version @{ collection_version }@
 
 {% endfor %}
 
+{% if 'role' in plugin_maps %}
+Role Index
```

Comment:
I wonder if this is the correct order. For a majority of collections, most users are looking for modules. In the docs survey results, we had at least one person say that they were confused by the `Plugin Index` heading and thought they were on the wrong page. For a collection that includes modules, other plugins, and roles, I would probably expect that to be the order of presentation (modules first, then plugins, then roles).

Reply:
I moved it to the bottom in c28c8e44e8ceb6b375eb10eadd91ec821a5dfaa9.

Maybe we should rename "Plugin Index" to "Module and Plugin Index"?

Reply:
That's a good idea. Or "Module and Other Plugin Index"? Modules really are just a type of plugin . . . but I don't know how important it is for people to know that . . .

Reply:
> That's a good idea. Or "Module and Other Plugin Index"? Modules really are just a type of plugin . . . but I don't know how important it is for people to know that . . .

+1 to changing to this.